File: python/types/strings/type_string.py
#!/usr/bin/python3

s = "hello"

print("string \"hello\": {} ({})\n".format(s, type(s)))

s = 'hello'

print("string 'hello': {} ({})\n".format(s, type(s)))

print("length: {}\n".format(len(s)))

s1 = "abc"
s2 = "def"
s3 = s1 + s2
print("{} + {} = {}\n".format(s1, s2, s3))

cite = "Stat rosa pristina nomine, nomina nuda tenemus."

print("cite: {}".format(cite))
print("words quantity: {}".format(len(cite.split())))

File: php/types/strings/type_string.php
#!/usr/bin/php8.0
<?php

$s = "hello";

echo "string \"hello\": $s (" . gettype($s) . ")\n";
print_r($s); echo "\n";
var_dump($s); echo "\n";

$s = 'hello';

echo "string 'hello': $s (" . gettype($s) . ")\n";
print_r($s); echo "\n";
var_dump($s); echo "\n";

print "length: " . strlen($s) . "\n";

echo "\n";

$s1 = "abc";
$s2 = "def";
$s3 = $s1 . $s2;
print "$s1 . $s2 = $s3\n";

echo "\n";

$cite = "Stat rosa pristina nomine, nomina nuda tenemus.";

print "cite: $cite\n";
print "words quantity: " . str_word_count($cite) . "\n";
